vueapp/src/components/Waiting.vue

```diff
@@ -1,16 +1,18 @@
 <template>
   <v-container fluid fill-height class="d-flex flex-column">
     Waiting List
-    <v-list>
-      <v-list-item v-for="player in store.waitingPlayers">{{
-        player.name
-      }}</v-list-item>
-    </v-list>
+    <draggable :list="store.waitingPlayers">
```

Comment:
How does the rearrange order of the items get reflected/persisted back to the store?

Reply:
Looks like the control mutates the bounded list.  In this case we'll have to reconsider how the player's store is designed.  Currently `allPlayers` is designed to hold all players in the session, including those playing (not in the waiting list).  The `waitingPlayers` list is just a dynamic filter of `allPlayers`.  We could simply remain `allPlayers` to `waitingPlayers` and treat it as such.